pybliometrics/scopus/superclasses/base.py

```diff
@@ -154,3 +165,42 @@ def _check_file_age(self):
         refresh = True
         mod_ts = None
     return refresh, mod_ts
+
+def _get_references(res: dict) -> list:
+    """Get references list in responce"""
+    return res.get('abstracts-retrieval-response', {}).get('references', {}).get('reference', [])
+
+def _get_all_refs(url: str,
+                  api: str,
+                  params: dict,
+                  verbose: bool,
+                  resp: dict,
+                  *args, **kwds) -> dict:
+    """Get all references"""
+    # startref starts at 1 (0 does not work)
+    # Max refs per query are 40
+    # Use of refcount leads to errors
+    res = resp.json()
+    n = int(res.get('abstracts-retrieval-response', {}).get('references', {}).get('@total-references', 0))
+    
+    # Initialize data
+    data  = res
```

Comment:
Seems superfluous

Reply:
`res` is used as a temporary auxiliary variable in the loop to parse the content of the API request. `data` saves the initial request with all its information. Just the relevant part of the API request (`res`) is saved in `data`.

Reply:
I see. Can you please add this in a short one-line comment behind the command?